test/performance/throughput/xml/throughput_intraprocess_best_effort_profile.xml

```diff
@@ -1,9 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <dds xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
-    <library_settings>
-        <intraprocess_delivery>USER_DATA_ONLY</intraprocess_delivery>
-    </library_settings>
```

Comment:
It has been moved at the end of the document.
Initially I moved it inside `<profiles>` tag (as a design decision), but then I realized that Fast DDS only supports it as it was. Then I restore all `<library_settings>` at the top of all their documents, but some tests failed because the first element read expected was a `<profiles>` instead of a `<library_settings>`, so most of them have been moved to the bottom of the document.